contactsummary/js/src/contact-summary/services/contract.js

```diff
@@ -175,7 +176,7 @@ define([
               });
           }
 
-          return promiseCache.getContractDetails;
+          return promiseCache[cacheKey];
```

Comment:
So this method returns promise, but it is documented that it returns `{*}`. Could you please change the documentation to the function that it returns a `{Promise}`?